### ISSUE TITLE
Add info about keyboard shortcuts when widget is selected

### ISSUE THREAD

**Issue by Dumluregn** [ckeditor/ckeditor4#4559]
## Type of report

Docs

## Provide a description of requested docs changes

Since we are improving the keyboard accessibility of block widgets (#4433 and #4467), it's a good opportunity to add missing info about widgets handling in [our docs article](https://ckeditor.com/docs/ckeditor4/latest/features/shortcuts.html). It can either be key-oriented (so e.g. add info for `Enter` key that it opens widget dialog) or widget-oriented (adding a new section describing what each key does when widget is selected) - the second version is clearer for me (see the equivalent [CKEditor 5 docs article](https://ckeditor.com/docs/ckeditor5/latest/features/keyboard-support.html#content-editing)).

Keyboard shortcuts to be mentioned:
- `left arrow` / `right arrow` - navigate before/after widget
- `enter` - open widget dialog
- `shift+alt+enter` / `shift+enter` - add paragraph before/after a widget.

Note that because of `enter` key associated to dialog we could not make it the same as in CKE5.

**Comment by f1ames** [comment]
Fixed in ckeditor/ckeditor4-docs#349.
